docs/docs/features.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 3
+title: Features
+---
+
+
+## Inspektor basic features
```

Comment:
can we merge this with installation because this steps are can be performed after installation.